asset_dashboard/static/js/components/PortfolioTable.js

```diff
@@ -1,20 +1,17 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import ReactTable from './BaseTable'
 import Button from 'react-bootstrap/Button'
 import Form from 'react-bootstrap/Form'
 import InputGroup from 'react-bootstrap/InputGroup'
 import projectColumns from './table_utils/projectColumns'
+import SubRow from './table_utils/SubRow'
 
 const PortfolioTable = ({ portfolio, onRemoveFromPortfolio, savePortfolioName, savePortfolio, createNewPortfolio }) => {
-  const [edit, setEdit] = useState(portfolio.name ? false : true)
-
-  const onRowClick = ({ original }) => {
```

Comment:
needed to remove this so the subrow click could open. before this code change, clicking the entire row would add/remove the row from the portfolio. this made it impossible to click the subrow dropdown button.